src/create-repo-relationship.ts

```diff
@@ -1,8 +1,38 @@
-export async function createRepoRelationships(depsList, mainRepo, j1Client, repoName, missingDeps: string[]) {
+// Transform the scope name if the what's in the repo (e.g. Github) does not
+// match what's in npm (other than the `@` sign).
+// For example, TitleCase vs. all lowercase below:
+// - in Github: `JupiterOne/jupiterone-client-nodejs`
+// - in npm: `@jupiterone/jupiterone-client-nodejs`
+const scopeNameTransformations = new Map([
+  ['@jupiterone/', 'JupiterOne/']
+])
+
+export async function createRepoRelationships(depsList: string[], mainRepo, j1Client, repoName, missingDeps: string[]) {
   const successFail = {success: 0, failure: 0, missingDeps: []};
+  const depRegex = /^(@\w+\/)?([^:]+)(:.*)?$/;
```

Comment:
I think I'm missing something... what is this regex supposed to match on / guard against?

Reply:
Nevermind, I see it is effectively splitting at the slash in something like `@jupiterone/package`.  I personally prefer for things that are simpler like this to just do `.split('/')` then do a check for `.startsWith('@')`

Reply:
the reason i am using regex is to kill two birds with one stone.  I also need to capture the scope and use it for potential transformation.

Reply:
I guess i could use split... will try that

Reply:
tried split.  i like the regex better :)